index.js

```diff
@@ -209,7 +209,13 @@ module.exports = {
     pkg.description = `Test app for ${this.locals(this.options).addonName} addon`;
 
     // we must explicitly add our own v2 addon here, the implicit magic of the legacy dummy app does not work
-    pkg.devDependencies[this.locals(this.options).addonName] = '^0.0.0';
+    if (useWorkspaceProtocol) {
+      // https://pnpm.io/workspaces#workspace-protocol-workspace
+      // For an app, `*` is fine, but if we were wiring this up to multiple packages, we'd want `^`
```

Comment:
I was thinking as well we could use * here, until I read this comment. Can you explain what you mean here?

Reply:
it is using `workspace:*`, but yarn and npm use `^0.0.0`.
I added to the "defaults" tests to demonstrate